src/_pytest/capture.py

```diff
@@ -692,6 +704,39 @@ def snap(self):
         return res
 
 
+class OrderedCapture(SysCapture):
+    """This class uses deque to keep streams order."""
+    streams = collections.deque()
+
+    def _get_writer_class(self):
+        return OrderedWriter(self.fd)
```

Comment:
Returns an object, not a class, doesn't it?

Reply:
right, the name is wrong